pkg/handlers/handlers.go

```diff
@@ -285,14 +285,9 @@ func HandleFile(
 	}
 	defer func() {
 		// Ensure all data is read to prevent broken pipe.
-		_, copyErr := io.Copy(io.Discard, rdr)
-		if copyErr != nil {
-			err = fmt.Errorf("error discarding remaining data: %w", copyErr)
-		}
-		closeErr := rdr.Close()
-		if closeErr != nil {
+		if closeErr := rdr.Close(); closeErr != nil {
 			if err != nil {
-				err = fmt.Errorf("%v; error closing reader: %w", err, closeErr)
+				err = fmt.Errorf("%w; error closing reader: %w", err, closeErr)
```

Comment:
Should this be updated to `errors.Join` too? Or keep only one wrapped? imo the double `%w` is misleading/confusing.

Reply:
Yea definitely. My stupid linter kept telling me to use %w, but that isn't a thing. 🤦